ff2mpv.js

```diff
@@ -1,14 +1,23 @@
+function onResponse(response) {
+    console.log("Received: " + response);
+}
+
+function onError(error) {
+    console.log(`${error}`);
+}
+
 function ff2mpv(url) {
     browser.tabs.executeScript({
-        code: "video = document.getElementsByTagName('video');video[0].pause();"
+        code: "video = document.getElementsByTagName('video');video[0].pause();",
     });
-    browser.runtime.sendNativeMessage("ff2mpv", { url: url });
+    var sending = browser.runtime.sendNativeMessage("ff2mpv", { url: url });
```

Comment:
Use a continuation style here instead of declaring a single use `sending` variable:

```js
browser.runtime.sendNativeMessage(...).then(onResponse, onError);
```

Reply:
Also, I'd prefer we remove `onResponse` entirely. In other words, just this:

```js
browser.runtime.sendNativeMessage(...).catch(onError);
```